components/TaskCard/index.js

```diff
@@ -26,6 +56,15 @@ export const TaskCard = ({ project, index }) => {
         <Label>Deadline: {projectDue}</Label>
         <Label>Size: {projectSize}</Label>
       </Labels>
+      <label>
+        <input
+          type="checkbox"
+          id="scales"
+          name="scales"
+          checked={checked}
+          onChange={onChangeCheckboxHandler}
+        />
+      </label>
     </TaskListItem>

```

Comment:
There are three labels in a row. `labels` belong to `inputs`, meaning that any one `label` should be connected to one `input`.
If you need to display additional info, use the `p` element for that. Otherwise, since there's only one `input` element, use only one `label`.

Reply:
thanks, let's try